.eslintrc.cjs

```diff
@@ -6,4 +6,11 @@ module.exports = {
   globals: {
     it: true,
   },
+  env: {
```

Comment:
Ultra nit: Turn this file into a .eslintrc (a JSON file) to match C-S-S and the extension; that way you don't need cjs :)